<launch>
	<node pkg="sunray_swarm" name="rmtt_station" type="rmtt_station" output="screen">
        <rosparam command="load" file="$(find sunray_swarm)/launch_rmtt/rmtt_params.yaml" />
	</node>
</launch>